#Repairnator properties
#Thu Apr 06 06:52:00 CEST 2017
step-durations=[CheckoutBuild\=3, \=0, ComputeClasspath\=9, CloneRepository\=29, PushIncriminatedBuild\=7, ComputeSourceDir\=0, SquashRepository\=7]
workspace=./workspace
pr-remote-repo=knoguchi/druid
pr-head-commit-id=8868be33e1392612b8ff13e54e41c497b2ac3424
pr-id=4039
repo=druid-io/druid
failingModule=/root/workspace/druid-io/druid/219101881/server
is-pr=true
buildid=219101881
error-types=java.lang.AssertionError
failing-test-cases=FailureLocation{className\='io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest', failingMethods\=[io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest\#testRetryAllFail, io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest\#testRetrySuccessAtSecondLocation], erroringMethods\=[], failures\=[FailureType{failureName\='java.lang.AssertionError', failureDetail\='null', isError\=false}, FailureType{failureName\='java.lang.AssertionError', failureDetail\='null', isError\=false}], nbFailures\=2, nbErrors\=0},FailureLocation{className\='io.druid.segment.loading.LocalDataSegmentPusherTest', failingMethods\=[io.druid.segment.loading.LocalDataSegmentPusherTest\#testPushCannotCreateDirectory], erroringMethods\=[], failures\=[FailureType{failureName\='java.lang.AssertionError', failureDetail\='Expected test to throw (an instance of java.io.IOException and exception with message a string containing "Unable to create directory")', isError\=false}], nbFailures\=1, nbErrors\=0},FailureLocation{className\='io.druid.query.lookup.LookupSnapshotTakerTest', failingMethods\=[io.druid.query.lookup.LookupSnapshotTakerTest\#testIOExceptionDuringLookupPersist], erroringMethods\=[], failures\=[FailureType{failureName\='java.lang.AssertionError', failureDetail\='Expected exception\: io.druid.java.util.common.ISE', isError\=false}], nbFailures\=1, nbErrors\=0}
pr-base-commit-id=dfe4bda7fd68be04712c612f9b027516e920c328
